src/Microsoft.Extensions.Options/OptionsServiceCollectionExtensions.cs

```diff
@@ -50,8 +63,11 @@ public static IServiceCollection Configure<TOptions>(this IServiceCollection ser
                 throw new ArgumentNullException(nameof(configureOptions));
             }
 
-            services.AddSingleton<IConfigureOptions<TOptions>>(new ConfigureOptions<TOptions>(configureOptions));
+            services.AddSingleton<IConfigureOptions<TOptions>>(new ConfigureOptions<TOptions>(name, configureOptions));
             return services;
         }
+
+        public static IServiceCollection ConfigureAll<TOptions>(this IServiceCollection services, Action<TOptions> configureOptions) where TOptions : class
```

Comment:
>ConfigureAll [](start = 41, length = 12)

What does ConfigurAll do? Also, looks like it will result in null ref exception as currently implemented.